### PR TITLE
added new workbook for app sign-in health

### PR DESCRIPTION
**Pull request by danielwood95**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Azure Active Directory/AppSigninHealth/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"App sign-in health",
+    "author": "Microsoft",
+    "isPreview": false,
```
Comment (KranthiKiranerusu) [inline Workbooks/Azure Active Directory/AppSigninHealth/settings.json:5]: why is this not a preview?

File: Workbooks/Azure Active Directory/AppSigninHealth/appsigninhealth.workbook [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "parameters - 3"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let thisWeek = SigninLogs\r\n| where TimeGenerated {time}\r\n| project TimeGenerated, AppDisplayName, UserPrincipalName, Status\r\n| where Status.errorCode == 0\r\n| where \"{Application}\" == \"All apps\" or AppDisplayName contains \"{Application}\"\r\n| summarize users = dcount(UserPrincipalName) by bin(TimeGenerated, 1hr)\r\n| sort by TimeGenerated desc\r\n| serialize rn = row_number();\r\nlet lastWeek = SigninLogs\r\n| where TimeGenerated between(({time:start} - totimespan(7d))..({time:end} - totimespan(7d)))\r\n| project TimeGenerated, AppDisplayName, UserPrincipalName, Status\r\n| where Status.errorCode == 0\r\n| where \"{Application:escape}\" == \"All apps\" or AppDisplayName contains \"{Application:escape}\"\r\n| summarize usersPriorWeek = dcount(UserPrincipalName) by bin(TimeGenerated, 1hr)\r\n| sort by TimeGenerated desc\r\n| serialize rn = row_number();\r\nthisWeek\r\n| join (lastWeek) on rn\r\n| project TimeGenerated, users, usersPriorWeek",
```
Comment (KranthiKiranerusu) [inline Workbooks/Azure Active Directory/AppSigninHealth/appsigninhealth.workbook:141]: please ensure to use has, has_cs etc whereever applicable

File: Workbooks/Azure Active Directory/AppSigninHealth/appsigninhealth.workbook [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "paragraph",
+        "links": [
+          {
+            "id": "5e7375d4-8e85-4fea-bb2f-605ddf4f7bf4",
+            "cellValue": "https://aka.ms/aadalertsdocs",
+            "linkTarget": "Url",
+            "linkLabel": "Learn more.",
+            "preText": "Monitor the number of successful sign-ins to your applications each hour compared to the same time the previous week. You can set automated alerts in case the number of successful sign-ins changes significantly, or if the failure rate exceeds a certain threshold.  ",
```
Comment (KranthiKiranerusu) [inline Workbooks/Azure Active Directory/AppSigninHealth/appsigninhealth.workbook:60]: this seems like a very targetted, 1 visualization workbook. I think we need to enhance the scope of this.

**Review by KranthiKiranerusu — CHANGES_REQUESTED** [review]

**Review by KranthiKiranerusu — CHANGES_REQUESTED** [review]
Please address the comments.

**Comment by danielwood95** [comment]
Per our discussion with Gloria and Kranthi this morning, publishing the App Sign-in Health workbook in a new Health menu area in the gallery. We will measure usage and remove if not adopted.

**Review by KranthiKiranerusu — APPROVED** [review]

**Review by KranthiKiranerusu — APPROVED** [review]

**Pull request merged**